sym/provider/flows_filter_resource.go

```diff
@@ -0,0 +1,139 @@
+package provider
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/symopsio/terraform-provider-sym/sym/client"
+	"github.com/symopsio/terraform-provider-sym/sym/utils"
+)
+
+// FlowsFilter Resource
+//
+// This resource allows customers to specify which flows can be shown to which users
+// using a Python `implementation` file
+func FlowsFilter() *schema.Resource {
+	return &schema.Resource{
+		Description:   "The `sym_flows_filter` resource provides a FlowsFilter that can be used to filter the Flows displayed to the requester. You may only have one FlowsFilter resource in your organization.",
+		CreateContext: createFlowsFilter,
+		ReadContext:   readFlowsFilter,
+		UpdateContext: updateFlowsFilter,
+		DeleteContext: deleteFlowsFilter,
+		Importer: &schema.ResourceImporter{
+			StateContext: getSlugImporter("flows_filter"),
+		},
+		Schema: map[string]*schema.Schema{
+			"implementation": {
+				Type:             schema.TypeString,
+				Required:         true,
+				ValidateDiagFunc: ImplementationValidation,
+				Description:      "Python code defining the `get_flows` reducer for the FlowsFilter.",
+			},
+			"vars":         utils.SettingsMap("A map of variables and their values to pass to this filter's Python implementation file."),
+			"integrations": utils.SettingsMap("A map of Integrations available when executing this FlowsFilter's implementation."),
+		},
```

Comment:
```suggestion
			"vars":         utils.SettingsMap("A map of variables and their values to pass to this FlowFilter's implementation."),
			"integrations": utils.SettingsMap("A map of Integrations available when executing this FlowsFilter's implementation."),
		},
```

nit: Since `integrations` uses `this FlowsFilter`, we should use the same for `vars` for consistency